template.tpl

```diff
@@ -19,8 +19,8 @@ ___INFO___
     "PERSONALIZATION"
   ],
   "brand": {
-    "id": "Didomi",
-    "displayName": "Didomi",
+    "id": "github.com_didomi",
```

Comment:
Why this change?

Reply:
That one got initially changed here https://github.com/didomi/gtm-template/commit/57c1f50f2ada3077859102ec1c526e8cd5affd05

It comes directly from the template
<img width="528" alt="Screenshot 2022-09-20 at 19 30 32" src="https://user-images.githubusercontent.com/30706016/191325812-45180af4-d1d5-4d86-ab65-7bfeb98c57a4.png">

I have been manually keeping the pair "id/displayName". Leaving it like this should help us to just copy/paste the TPL file export and not modify this field.